vllm/model_executor/models/gpt_neox.py

```diff
@@ -54,6 +54,8 @@ def __init__(
         self.total_num_heads = config.num_attention_heads
         self.hidden_size = config.hidden_size
         self.head_size = self.hidden_size // self.total_num_heads
+        self.bias = "True" if not hasattr(
+            config, "attention_bias") else config.attention_bias
```

Comment:
Can we shorten this like the following?
```suggestion
        self.bias = getattr(config, "attention_bias", True)
```

Reply:
Looks better to me. thanks for suggestion!